docs/language.js

```diff
@@ -1155,7 +1155,7 @@ var data = [
 		"output" : "*",
 		"permissions" : "e",
 		"new value" : "new",
-		"description" : "Evaluates to code that, if evaluated, would completely reproduce the entity specified by id, as well as all contained entities.  If include_rand_seeds is true, its default, it will include all entities' random seeds.  If parallel_create is true, then the creates will be performed with parallel markers as appropriate for each group of contained entities.",
+		"description" : "Evaluates to code that, if called, would completely reproduce the entity specified by id, as well as all contained entities.  If include_rand_seeds is true, its default, it will include all entities' random seeds.  If parallel_create is true, then the creates will be performed with parallel markers as appropriate for each group of contained entities.  The code returned accepts two parameters, create_new_entity, which defaults to true, and new_entity, which defaults to null.  If create_new_entity is true, then it will create a new entity with id specified by new_entity, where null will create an unnamed entity.  If create_new_entity is false, then it will overwrite the current entity's code and create all contained entities.",
```

Comment:
i suggest adding an example line where you demonstrate these new parameters, eg:
`(call fe (assoc create_new_entity (false) new_entity "bobzilla"))`